src/lib/slotMachine.ts

```diff
@@ -1,29 +1,25 @@
 import { sum } from './hashingFunctions';
 
 class SlotMachine<T> {
-  private slots: T[];
   private numSlots: number;
-  private hashingFn: (items: number[]) => number;
 
-  constructor(slots: T[], hashingFn?) {
-    this.slots = slots;
-    this.numSlots = this.slots.length;
-    this.hashingFn = hashingFn || sum;
+  constructor(private slots: T[], private hash = sum) {
```

Comment:
i see what the `private slots/hash` does but took me a minute to realize it. Neat!